beacon_node/src/cli.rs

```diff
@@ -188,6 +188,13 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true)
                 .default_value("http://localhost:8545")
         )
+        .arg(
+            Arg::with_name("slots-per-restore-point")
+                .long("slots-per-restore-point")
+                .value_name("SLOT_COUNT")
+                .help("Specifies how often a freezer DB restore point should be stored.")
```

Comment:
Added a default, and a warning that this flag shouldn't be tampered with lightly. Opened #673 to address the issue of cleanly notifying the user that they've changed `slots_per_restore_point` inappropriately. Currently it will error as soon as you interact with the freezer DB, with an error about missing restore points, unless you only increase the number of slots per restore point -- that might be fine.